src/cholesky/lkjcholesky.jl

```diff
@@ -40,7 +40,7 @@ end
 #  Constructors
 #  -----------------------------------------------------------------------------
 
-function LKJCholesky(d::Int, η::Real, _uplo::Union{Char,Symbol} = 'L'; check_args = true)
+function LKJCholesky(d::Int, η::Real, _uplo::Union{Char,Symbol} = 'L'; check_args::Bool=true)
```

Comment:
Help me understand, what effect does this have?

Reply:
It ensures that one can only pass `true` or `false` for `check_args`, and hence no weird or surprising things can happen in the `if check_arg` statement in the function body. There's no breaking change since passing anything else would be a bug, it just makes the implementation a bit safer.

Reply:
Ok, so this is just for consistency